### PR TITLE
Change format input operation

### PR DESCRIPTION
**Pull request by AurelienFT**
The `send_operations` API endpoint now have this format as input : 
```rust
#[derive(Serialize, Deserialize, Debug)]
pub struct OperationInput {
    pub creator_public_key: PublicKey,
    pub signature: Signature,
    pub serialized_content: String,
}
```

### CODE REVIEW

**Review by Eitu33 — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

File: docs/technical-doc/api.rst [hunk 1]
```diff
@@ -478,11 +489,9 @@ pool.
                 "gas_price": Number, // Amount
             }
         },
-        "sender_public_key": String
-        },
-        "signature": String
     }
-    ]]
+
+For the signature you need to use the bytes of the public_key and content in byte compact concatenated and sign it with blake3.
```
Comment (qdrn) [inline docs/technical-doc/api.rst:494]: Blake3 is a hashing algorithm, not a signature algorithm. cc @AurelienFT

Comment (AurelienFT) [reply docs/technical-doc/api.rst:494]: Ah yes it's schnorr my bad. Will change it

Comment (AurelienFT) [reply docs/technical-doc/api.rst:494]: Done in #2744

**Pull request merged**